management-model/src/main/java/org/terracotta/management/stats/NumberUnit.java

```diff
@@ -20,7 +20,7 @@
  */
 public enum NumberUnit {
 
-  PERCENT {
+  RATIO {
```

Comment:
what's the motive for this renaming ?
also, percentage should have been renamed as well